chapter4.adoc

```diff
@@ -347,8 +347,8 @@ batch optimized via the use of the sequence `SFENCE.W.INVAL`, `MINVAL.SPA` and
 
 [NOTE]
 ====
-MTT lookups that began while `mttp` was active are not required to complete or
-terminate when `mttp` is no longer active, unless a `MFENCE.SPA` instruction
+Ongoing MTT lookups are not required to complete or terminate when `mttp` is 
+effectively disabled, unless a `MFENCE.SPA` instruction
```

Comment:
```suggestion
MTT lookups that began while `mttp` was active are not required to complete or
terminate when `mttp` is no longer active, unless a `MFENCE.SPA` instruction
```

Reply:
reverting this change - following the same language as the priv spec for vs and g stage lookups.